infra/main.bicep

```diff
@@ -90,8 +90,9 @@ module ai 'core/host/ai-environment.bicep' = {
       ? storageAccountName
       : '${abbrs.storageStorageAccounts}${resourceToken}'
     openAiName: !empty(openAiName) ? openAiName : 'aoai-${resourceToken}'
+    openAiConnectionName: 'aoai-connection-${resourceToken}'
```

Comment:
@nitya  is this related to https://github.com/Azure-Samples/contoso-chat/issues/137?